pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/column/PhysicalColumnIndexContainer.java

```diff
@@ -133,7 +133,11 @@ public PhysicalColumnIndexContainer(SegmentDirectory.Reader segmentReader, Colum
       _rangeIndex = null;
     }
 
-    PinotDataBuffer fwdIndexBuffer = segmentReader.getIndexFor(columnName, ColumnIndexType.FORWARD_INDEX);
+    // Setting the 'fwdIndexBuffer' to null if forward index is enabled. No-op index readers will be setup for the
```

Comment:
(nit) `typo enabled -> disabled` 

> No-op index readers will be setup for the forward index disabled columns ...

This is no longer the case I guess since we rely on throwing exceptions. So might want to change the comment.

Reply:
good catch, done